### PR TITLE
feat(IQFeed/ToolBox): add IQFeed support for docker

### PR DESCRIPTION
**Pull request by mathpaquette**
#4631

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which improves implementation)
- [ ] Performance (non-breaking change which improves performance. Please add associated performance test and results)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [ ] I have added tests to cover my changes. 
- [ ] All new and existing tests passed.
- [ ] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

**Comment by mathpaquette** [comment]
image

**Comment by mathpaquette** [comment]
@jaredbroad it also requires my other PR to work properly because I fixed some parsing issues with some of the IQFeed symbols.

File: run_docker_iqfeed.bat [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+REM you may not use this file except in compliance with the License.
+REM You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+REM
+REM Unless required by applicable law or agreed to in writing, software
+REM distributed under the License is distributed on an "AS IS" BASIS,
+REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+REM See the License for the specific language governing permissions and
+REM limitations under the License.
+
+set docker_params=--expose 5009^
```
Comment (mathpaquette) [inline run_docker_iqfeed.bat:16]: @jaredbroad not sure these are mandatory but at least now we have the ability the pass them in.

File: ToolBox/IQFeed/IQFeedDataQueueHandler.cs [hunk 2]
```diff
@@ -219,11 +219,16 @@ private void Connect()
         {
             try
             {
-                //Launch the IQ Feed Application:
+                // Launch the IQ Feed Application:
                 Log.Trace("IQFeed.Connect(): Launching client...");
 
-                var connector = new IQConnect(Config.Get("iqfeed-productName"), "1.0");
-                connector.Launch();
+
```
Comment (C-SELLERS) [inline ToolBox/IQFeed/IQFeedDataQueueHandler.cs:225]: extra space

File: run_docker.bat [hunk 3]
```diff
@@ -87,6 +87,12 @@ set command=docker run --rm --mount type=bind,source=%config_file%,target=/Lean/
     --name LeanEngine^
     -p 5678:5678
 
+REM If docker_params exist, add them to docker command
+if not "%docker_params%" == "" (
+    set command=%command% %docker_params%
+    echo Applying additionnal docker parameters to docker command
```
Comment (C-SELLERS) [inline run_docker.bat:93]: Typo

File: run_docker_iqfeed.bat (hunk 1 above)
Comment (C-SELLERS) [reply run_docker_iqfeed.bat:16]: Lets find out which ones we need for sure and scrap the rest

Comment (mathpaquette) [reply run_docker_iqfeed.bat]: guys, let's try without any, should be working fine as well. at least we gonna have the ability to add some parameters if needed like --net=host for those who require it.

File: ToolBox/IQFeed/readme.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Setup IQFeed 
+
+## 1. Sign up for [IQFeed](https://www.iqfeed.net/trent/index.cfm?displayaction=start&promo=1996499)
```
Comment (mathpaquette) [inline ToolBox/IQFeed/readme.md:3]: here @jaredbroad

**Pull request merged**